blobstore/src/main/java/org/jclouds/blobstore/LocalStorageStrategy.java

```diff
@@ -114,14 +114,18 @@
      */
     Blob getBlob(String containerName, String blobName);
 
+    @Deprecated
+    String putBlob(String containerName, Blob blob) throws IOException;
```

Comment:
Should this not have been

```java
default String putBlob(String containerName, Blob blob) throws IOException {
    return putBlob(containerName, blob, BlobAccess.PRIVATE);
}
```

?

Reply:
I think it could have been although this would be the first default method in jclouds since 2.3.0 was the first version to require Java 8.